<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDoctorDialog</class>
 <widget class="QDialog" name="AddDoctorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Врачи - Добавление</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayoutDataForm">
     <item row="1" column="0">
      <widget class="QLabel" name="labelSpeciality">
       <property name="text">
        <string>Специальность</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelPhone">
       <property name="text">
        <string>Номер телефона</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelFullname">
       <property name="text">
        <string>ФИО</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelExperience">
       <property name="text">
        <string>Стаж</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditFullname"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditSpeciality"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditPhone"/>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBoxExperience">
       <property name="maximum">
        <number>40</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutAction">
     <item>
      <widget class="QPushButton" name="pushButtonAdd">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
